packages/edit-site/src/components/sidebar-navigation-screen-templates/index.js

```diff
@@ -46,6 +46,22 @@ const config = {
 				'Manage what patterns are available when editing your site.'
 			),
 		},
+		sortCallback: ( items ) => {
+			const groupedByArea = items.reduce(
+				( accumulator, item ) => {
+					const key = accumulator[ item.area ] ? item.area : 'rest';
+					accumulator[ key ].push( item );
+					return accumulator;
+				},
+				{ header: [], footer: [], sidebar: [], rest: [] }
```

Comment:
Is the `sidebar` template part still a thing? 🤔

Reply:
It has never been exposed in the UI, except perhaps if a theme supplies one?